src/Symfony/Component/HttpKernel/EventListener/ErrorListener.php

```diff
@@ -33,8 +33,14 @@ class ErrorListener implements EventSubscriberInterface
     protected $controller;
     protected $logger;
     protected $debug;
+    /**
+     * @var array<class-string, array{log_level: string|null, status_code: int<100,599>|null}>
+     */
     protected $exceptionsMapping;
 
+    /**
+     * @param array<class-string, array{log_level: string|null, status_code: int<100,599>|null}> $exceptionsMapping
```

Comment:
Is it welcome to explicitly specify typing for mapping structure in Symfony codebase for now?